raiden/smart_contracts/NettingChannelLibrary.sol

```diff
@@ -623,4 +630,8 @@ library NettingChannelLibrary {
             n[i-start] = a[i];
```

Comment:
I know you did not touch this, but I can now see that a space is needed between `-` and its surroundings.